<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="21855" started-at="2021-02-05T06:00:11Z" finished-at="2021-02-05T06:00:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="21855" started-at="2021-02-05T06:00:11Z" finished-at="2021-02-05T06:00:33Z">
      <class name="training.selenium.session10.testscripts.FailSSTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:training.selenium.session10.testscripts.FailSSTest@6f7fd0e6]" name="setup" is-config="true" duration-ms="11710" started-at="2021-02-05T06:00:11Z" finished-at="2021-02-05T06:00:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="validateLoginTest()[pri:0, instance:training.selenium.session10.testscripts.FailSSTest@6f7fd0e6]" name="validateLoginTest" duration-ms="7768" started-at="2021-02-05T06:00:23Z" finished-at="2021-02-05T06:00:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Adactin.com - Search Hotel] but found [Adactin.com - Hotel Reservation System]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Adactin.com - Search Hotel] but found [Adactin.com - Hotel Reservation System]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at training.selenium.session10.SearchHotelPage.verifyTitle(SearchHotelPage.java:19)
at training.selenium.session10.testscripts.FailSSTest.validateLoginTest(FailSSTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:training.selenium.session10.testscripts.FailSSTest@6f7fd0e6]" name="tearDown" is-config="true" duration-ms="2344" started-at="2021-02-05T06:00:31Z" finished-at="2021-02-05T06:00:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validateLoginTest status=FAILURE method=FailSSTest.validateLoginTest()[pri:0, instance:training.selenium.session10.testscripts.FailSSTest@6f7fd0e6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- training.selenium.session10.testscripts.FailSSTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
